src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -139,7 +139,7 @@ interface ChannelProviderInterface extends ChannelContextProps, MessageStoreInte
   setIsScrolled?: React.Dispatch<React.SetStateAction<boolean>>;
   messageActionTypes: Record<string, string>;
   messagesDispatcher: CustomUseReducerDispatcher;
-  quoteMessage: SendableMessageType;
+  quoteMessage: SendableMessageType | null;
```

Comment:
this changes seems to be applied to `index_d_ts`

Reply:
Thanks for the catch! oh god, this manual process is so error prone..